libdnf/base/Base.hpp

```diff
@@ -47,28 +47,28 @@ class Base {
     // ACTIONS
 
     /// @replaces dnf:dnf/base.py:method:Base.autoremove(self, forms=None, pkg_specs=None, grp_specs=None, filenames=None)
-    void auto_remove(const Demands & demands, std::vector<std::string> patterns, std::vector<std::string> exclude_patterns);
+    void auto_remove(const Demands & demands, std::vector<std::string> patterns, std::vector<std::string> excludePatterns);
```

Comment:
libdnf/base/Base.hpp:50:107: warning: invalid case style for parameter 'excludePatterns' [readability-identifier-naming]
    void auto_remove(const Demands & demands, std::vector<std::string> patterns, std::vector<std::string> excludePatterns);
                                                                                                          ^~~~~~~~~~~~~~~
                                                                                                          exclude_patterns